source/base/parameter_handler.cc

```diff
@@ -1976,6 +2007,34 @@ ParameterHandler::scan_line(std::string        line,
             entries->get<std::string>(path + path_separator + "alias"));
         }
 
+      // Check if the entry (or the resolved alias) is deprecated, print a
```

Comment:
Could we restructure the code as in https://github.com/dealii/dealii/blob/30614176795cc91fe0b8999408397dbf5f349d34/include/deal.II/base/parameter_handler.h#L1659-L1660? Back when we introduced the function, we did agree on such an interface.